jina/types/sets/document.py

```diff
@@ -155,21 +165,34 @@ def all_contents(self) -> Tuple['np.ndarray', 'DocumentSet']:
         """
         return self._extract_docs('content')
 
-    def _extract_docs(self, attr: str) -> Tuple['np.ndarray', 'DocumentSet']:
-        contents = []
+    def _extract_docs(self, *attr: str) -> Tuple['np.ndarray', 'DocumentSet']:
+        list_of_contents_output = len(attr) > 1
+        if list_of_contents_output:
```

Comment:
Do you need this `if` `else` ?

Can't you simply do...
```
     contents = [[] for at in attr]
```

if `attr` is not  present then `contents = []`
If attr has 3 elements then `contents = [[],[],[]]`

Reply:
Good!

Reply:
If there is one element, it does `[[]]` which is what I want to avoid

Reply:
You can do that filtering 
```
    contents = [[] for at in attr if len(attr)>1]
```

Reply:
Done!